airbyte-integrations/connectors/source-bamboo-hr/components.py

```diff
@@ -53,94 +57,3 @@ def convert_custom_reports_fields_to_list(self, custom_reports_fields: str) -> l
     def _union_schemas(self, schema1, schema2):
         schema1["properties"] = {**schema1["properties"], **schema2["properties"]}
         return schema1
-
-
-CUSTOM_REPORTS_BASE_SCHEMA = {
```

Comment:
I moved the static schema to inline schema in the manifest.yaml and use `default_schema_loader` schema loader to load it here